core/domain/config_domain.py

```diff
@@ -102,6 +102,8 @@ class ConfigProperty(object):
     """
 
     def refresh_default_value(self, default_value):
+        """Refreshes the default_value of the configuration property."""
```

Comment:
Refreshes the configuration property to the default value.

Reply:
Done! @kevinlee12